setup.py

```diff
@@ -20,10 +20,15 @@
 """Setup the Hveto package
 """
 
+import sys
 from setuptools import setup
 
 import versioneer
 
+# enforce python version
+if sys.version < '3.5':
+    raise ImportError("Python versions older than 3.5 are not supported.")
+
```

Comment:
I would remove this block, the classifiers and `python_requires` statements are enough.